pom.xml

```diff
@@ -209,6 +209,11 @@
         <artifactId>jackson-xc</artifactId>
         <version>${codehaus.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.codehaus.jettison</groupId>
+        <artifactId>jettison</artifactId>
+        <version>1.5.1</version>
```

Comment:
this one has vuln: This link says fixed in https://app.snyk.io/org/snowflakedb-sca-scanning-public-repo/pr-checks/f45ef4b1-66a1-43ba-9cee-24d06a77157e 1.5.2

Reply:
Ack.